libs/dataset_deployer.py

```diff
@@ -0,0 +1,94 @@
+import os
+import io
+import logging
+import boto3
+
+_logger = logging.getLogger(__name__)
```

Comment:
what do you think if this lives in api or some persistance folder?

Reply:
I don't think there is much advantage to having the functionality for deploying live in the "api" folder just yet. For instance, I'm not sure exactly what logic will actually live in the API folder.  Some of that logic I think belongs in pipelines? If the API folder was just the schema objects that would make sense to me.

Until the folder more completely encapsulate the behavior, I'd prefer not creating them?

I saw something once that you don't need a folder until there are more than 8 files in a directory and I kinda like.  Once a folder starts getting overloaded i think you can identify similar functionality a bit easier.

Reply:
Noted.